website/config/navbar.js

```diff
@@ -78,6 +78,10 @@ module.exports = [
         to: "/docs/general/community",
         label: "Community"
       },
+      {
+        to: "/docs/general/events",
+        label: "Events"
+      },
```

Comment:
Is it necessary to put Events in the navigation, where each item is usually on a different page? 🤔

Reply:
> each item is usually on a different page? 

Hi, could you share more details on this? I'm not clear what do you mean 😅

Reply:
![image](https://user-images.githubusercontent.com/72343596/160040080-654641c0-b9d0-40c3-bf25-700fd53b58df.png)
Sorry, I meant that Community and Events are different categories on the same page. So, do we need to add events to the navigation? 😯

Reply:
After a quick confirmation with @guoqqqi, Qi means why we need to put the `Events` and `Community` menu in the same category dropdown, 😅 bcoz when users visit Community, they will also notice Events. 

We think it's ok to keep the current state.